src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,65 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import React from 'react';
+import data from '../../public/api/people.json';
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+const getSlugByName = (name: string) => {
+  const person = data.find(p => p.name === name);
```

Comment:
The `getSlugByName` function searches through the entire dataset each time it's called. This could be inefficient if the dataset is large. Consider optimizing this by using a more efficient data structure or caching the results.